includes/discovery/sensors/temperature/cmts.inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+$oids = snmp_walk($device, ".1.3.6.1.4.1.4998.1.1.10.1.4.2.1.29", "-Osqn", "");
```

Comment:
Can you not use one of the  snmpwalk_* functions? Saves doing a lot of post work on the data